xmodule/js/src/video/036_video_social_sharing.js

```diff
@@ -29,49 +30,24 @@
             };
 
             VideoSocialSharingHandler.prototype = {
+
                 // Initializes the module.
                 initialize: function() {
                     this.el = this.container.find('.wrapper-social-share');
                     this.el.on('click', '.btn-link', this.clickHandler);
                     this.baseVideoUrl = this.el.data('url');
                     this.course_id = this.container.data('courseId');
                     this.block_id = this.container.data('blockId')
-                    this.socialSharingSites = this.getSocialSharingSites()
                 },
 
+                // Fire an analytics event on share button click.
                 clickHandler: function(event) {
                     var self = this;
-                    event.preventDefault();
                     var source = $(event.currentTarget).data('source')
-                    var utmQuery = $.param({
```

Comment:
I believe these are refactored away into the new js file, right?

Reply:
This gets constructed on the backend and added to the template via: https://github.com/openedx/edx-platform/blob/af96cb1b213cab61968b7c5fc2f518e109a02edc/xmodule/video_block/sharing_sites.py#L73-L80